wpimath/util/units.py

```diff
@@ -0,0 +1,171 @@
+# the WPILib BSD license file in the root directory of this project.
+
+from math import degrees, radians, pi
+
+kInchesPerFoot = 12.0
+kMetersPerInch = 0.0254
+kSecondsPerMinute = 60
+kMillisecondsPerSecond = 1000
+kKilogramsPerLb = 0.453592
+
+
+def metersToFeet(meters: float) -> float:
+    """Converts given meters to feet.
+
+    :param meters: The meters to convert to feet.
+
+    :returns: Feet converted from meters.
+    """
+    return metersToInches(meters) / kInchesPerFoot
+
+
+def feetToMeters(feet: float) -> float:
+    """Converts given feet to meters.
+
+    :param feet: The feet to convert to meters.
+
+    :returns: Meters converted from feet.
+    """
+    return inchesToMeters(feet * kInchesPerFoot)
+
+
+def metersToInches(meters: float) -> float:
+    """Converts given meters to inches.
+
+    :param meters: The meters to convert to inches.
+
+    :returns: Inches converted from meters.
+    """
+    return meters / kMetersPerInch
+
+
+def inchesToMeters(inches: float) -> float:
+    """Converts given inches to meters.
+
+    :param inches: The inches to convert to meters.
+
+    :returns: Meters converted from inches.
+    """
+    return inches * kMetersPerInch
+
+
+def degreesToRadians(degrees: float) -> float:
+    """Converts given degrees to radians.
+
+    :param degrees: The degrees to convert to radians.
+
+    :returns: Radians converted from degrees.
+    """
+    return radians(degrees)
+
+
+def radiansToDegrees(radians: float) -> float:
+    """Converts given radians to degrees.
+
+    :param radians: The radians to convert to degrees.
+
+    :returns: Degrees converted from radians.
+    """
+    return degrees(radians)
+
+
+def radiansToRotations(radians: float) -> float:
+    """Converts given radians to rotations.
+
+    :param radians: The radians to convert.
+
+    :returns: rotations Converted from radians.
+    """
+    return radians / (pi * 2)
+
+
+def degreesToRotations(degrees: float) -> float:
+    """Converts given degrees to rotations.
+
+    :param degrees: The degrees to convert.
+
+    :returns: rotations Converted from degrees.
+    """
+    return degrees / 360
+
+
+def rotationsToDegrees(rotations: float) -> float:
+    """Converts given rotations to degrees.
+
+    :param rotations: The rotations to convert.
+
+    :returns: degrees Converted from rotations.
+    """
+    return rotations * 360
+
+
+def rotationsToRadians(rotations: float) -> float:
+    """Converts given rotations to radians.
+
+    :param rotations: The rotations to convert.
+
+    :returns: radians Converted from rotations.
+    """
+    return rotations * 2 * pi
+
+
+def rotationsPerMinuteToRadiansPerSecond(rpm: float) -> float:
+    """Converts rotations per minute to radians per second.
+
+    :param rpm: The rotations per minute to convert to radians per second.
+
+    :returns: Radians per second converted from rotations per minute.
+    """
+    return rpm * pi / (kSecondsPerMinute / 2)
```

Comment:
Given this is using float division anyway, writing it like this is just confusing. Use `math.tau` instead for these two conversations as well.